### PR TITLE
library: Ported `Spinner` in RUST

### PR DESCRIPTION
**Pull request by onkarrai06**
Ported the RUST implementation of Spinner in RUST, please check it out.

Also updated about.js as told on my first PR.

### CODE REVIEW

File: src/Library/demos/Spinner/code.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+use crate::workbench;
+use gtk::prelude::*;
+
+pub fn main() {
+    let button: gtk::Button = workbench::builder().object("button").unwrap();
+    let spinner: gtk::Spinner = workbench::builder().object("spinner").unwrap();
+
+    button.connect_clicked(move |button| {
+        if spinner.is_spinning() == true {
```
Comment (Hofer-Julian) [inline src/Library/demos/Spinner/code.rs:9]:
```suggestion
        if spinner.is_spinning() {
```

**Comment by onkarrai06** [comment]
Changed, please check.
@Hofer-Julian

File: src/Library/demos/Spinner/code.rs (hunk 1 above)
Comment (onkarrai06) [reply src/Library/demos/Spinner/code.rs]: Resolved, please check.

**Review by Hofer-Julian — APPROVED** [review]

**Comment by Hofer-Julian** [comment]
Thank you!

**Pull request merged**